support/docker/dev/Dockerfile

```diff
@@ -15,10 +15,22 @@ WORKDIR /home/user/PeerTube
 # Configure Cloud9 IDE to use PeerTube's source directory as workspace (-w).
 RUN sudo sed -i "s/-w \/home\/user/-w \/home\/user\/PeerTube/" /etc/supervisord.conf
 
+# Install dependencies.
+RUN yarn install
+
 # Configure Janitor for PeerTube.
 ADD janitor.json /home/user/
 RUN sudo chown user:user /home/user/janitor.json
 
 # Configure and build PeerTube.
-RUN yarn install \
- && npm run build
+ADD create_user.sql /tmp/
+RUN sudo service postgresql start && \
+    sudo -u postgres psql --file=/tmp/create_user.sql && \
+    sudo rm /tmp/create_user.sql
+
+ADD supervisord.conf /tmp/supervisord-extra.conf
+RUN cat /tmp/supervisord-extra.conf | sudo tee -a /etc/supervisord.conf && \
+    sudo rm /tmp/supervisord-extra.conf
+
+EXPOSE 3000
+EXPOSE 9000
```

Comment:
Nit: You can expose multiple ports in one instruction [like so](https://github.com/JanitorTechnology/dockerfiles/blob/6d3e7835c90fafe76a2e8463b4d5104ac0a39a9d/ubuntu-dev/ubuntu-dev.dockerfile#L224).